resources/assets/less/bem/profile-stats.less

```diff
@@ -95,5 +126,10 @@
       text-align: center;
       padding-top: 10px;
     }
+
+    &--level-progress {
+      letter-spacing: 1px;
```

Comment:
?

Reply:
it feels like it's more spaced out than the other things in the design